js/about/newtab.js

```diff
@@ -96,6 +96,7 @@ class NewTabPage extends React.Component {
     }).size > 0
   }
   isBookmarked (siteProps) {
+    // XXX: Fixme, not passing state in!
```

Comment:
Could you check how often this is called? If it is often could you put this in a different component passing in only the URL? It would ensure each is only rendered / called once.